Sources/Client/Client/Client+WebSocket.swift

```diff
@@ -53,6 +53,7 @@ extension Client {
             
             if case .connected(let userConnection) = connectionState {
                 self.userAtomic.set(userConnection.user)
+                self.restoreWatchingChannels()
```

Comment:
@buh shouldn't you only call this if the connection broke in the meantime?

Reply:
I also recommend creating a `connectionRecovery` method and call it here (and ofc, connectionRecovery will do what restoreWatchingChannels does / call the same function)

Reply:
We have a reconnection flow.
This is the first point when the WebSocket is connected and everything will start working again.
Probably, I missed your point.